subsystems/elevatorsubsystem.py

```diff
@@ -93,7 +107,8 @@ def periodic(self) -> None:
                 self.setElevatorMotorsAtPosition(constants.kIntakePositionBeltPosition)
 
         self.elevatorStatePublisher.set(str(self.state))
-        self.elevatorPositionPublisher.set(self.getElevatorPosition())
+        if self.state != self.ElevatorState.ManualMode:
```

Comment:
replace != with is not, technically more python efficient since its a static value